workflows/genomeassembly.nf

```diff
@@ -347,7 +350,7 @@ workflow GENOMEASSEMBLY {
                        PREPARE_INPUT.out.busco,
                        GENOMESCOPE_MODEL.out.hist,
                        GENOMESCOPE_MODEL.out.ktab
-    )
+    )*/
```

Comment:
Was it commented out with the purpose of debugging and does it need uncommenting?